### DISCUSSION TITLE
Xml report from Gauge framework with Allure

### DISCUSSION THREAD

**Discussion by lirany1** [allure-framework/allure2#1881] [Questions & Support]
Hi :) 
i have a xml file ( junit5 form) that is a result of test in Gauge framework

using this command
 `allure -v serve build/reports/xml-report`
 i can see the report locally 

The question is how can i generate the Allure report using the that xml file through Jenkins as a result of a test stage in a job?

10x

**Comment by cheshi-mantu** [comment]
@lirany1 hi there!
have you [this plug-in](https://plugins.jenkins.io/allure-jenkins-plugin/) installed?

**Reply by lirany1** [reply]
@cheshi-mantu Hey mate
yes i did
i succeeded to convert the xml report to be as a Allure report in jenkins
now i want to ask how can i see also in the report the executors and trend that are empty
i am using a jenkins as a k8s so the changes should be inside the container

**Reply by cheshi-mantu** [reply]
pls refer to this description: https://docs.qameta.io/allure-report/#_history_files, although the history is limited for the report. If you require the historical data, you maybe need to think about [Allure TestOps](https://www.youtube.com/playlist?list=PLINOSjry419nVTsmiQ4ACukWDioU4QK2R)
